app/jobs/cangaroo/base_job.rb

```diff
@@ -0,0 +1,35 @@
+module Cangaroo
+  class BaseJob < Job
+    include Cangaroo::ClassConfiguration
+
+    class_configuration :connection
+    class_configuration :path, ''
+    class_configuration :parameters, {}
+    class_configuration :process_response, true
```

Comment:
@AlessioRocco `process_response` is only applicable for push jobs, not poll jobs. Might be more self-documenting to just include this option on the `PollJob`?